src/WobblySpinner.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { propTypes, defaultProps } from "./WobblySpinner.props";
+import * as wobblySpinnerSheet from "./wobblySpinnerSheet";
+
+function mountSheet(props) {
+    var { key, className } = wobblySpinnerSheet.mount(props);
+    this._key = key;
+    this._className = className;
+}
+
+export default class WobblySpinner extends React.Component {
+    static propTypes = propTypes
+    static defaultProps = defaultProps
+
+    componentWillMount() {
+        this::mountSheet(this.props);
```

Comment:
You know that I've never liked pseudo "private" methods by convention (prefixing with "_"). If they are private, then make them truly private. This accomplishes that.

As far as "::" bind, it's just less verbose that `.call` or `.bind`.